Update Contributing.md (#3213)
adoptium/aqa-tests: Contributing.md

## Developer resources

The AQAvit project maintains the following source code repositories:

* https://github.com/adoptium/aqa-tests - the central project for AQAvit
* https://github.com/adoptium/TKG - a lightweight test harness for running a diverse set of tests or commands
* https://github.com/adoptium/aqa-systemtest - system verification tests
* https://github.com/adoptium/aqa-test-tools - various test tools that improve workflow
* https://github.com/adoptium/STF - system Test Framework
* https://github.com/adoptium/bumblebench - microbenchmarking test framework

> * https://github.com/adoptium/run-aqa - run aqa GitHub action

Reviewer: ```suggestion
* https://github.com/adoptium/run-aqa - run-aqa GitHub action
```

For more information, please see the Eclipse Committer Handbook:

> https://www.eclipse.org/projects/handbook/#resources-commit

Reviewer: For ECA, we could additionally reference this guide, which gives more details and is more helpful than the Eclipse resources: https://github.com/adoptium/documentation/blob/main/documentation-documentation/eca-sign-off.adoc

git checkout -b my_new_branch
```

4. Commit once you are done with your work. 
```
git commit -s -m "message about this PR"
```

Commit message example:

```
Update jtreg in openjdk test

This patch updates jtreg version from 5.1 to 6.1.

Fixes: #1234

Signed-off-by: Full Name <email>
```

- The first line is the PR title. It should describe the change made. Please keep it short and simple.
- The body should include detailed information about your PR. You may want to include designs, rationale, and a brief explanation of what you have changed. Please keep it concise.
- When a commit has related issues, please use the Git keywords to automatically close or relate to the issues.
https://help.github.com/articles/closing-issues-using-keywords/
- Please sign off on your commit in the footer. This can be automatically added to your commit by passing `-s` to `git commit` (see above example).

5. Push the change into your Git repo:
```
git push origin my_new_branch
```

6. We would like to encourage you to open a pull request early and use `Create draft pull request` option. This allows others to check the PR, get early feedback, and helps create a better end product.

> 7. Run Grinder to validate your PR. Your can find recordings about how to use Grinder in AQA Lightning Talk Series: https://github.com/eclipse-openj9/openj9/wiki/AQA-Lightning-Talk-Series. 

Reviewer: ```suggestion
7. If you have been given access to run test jobs in our Jenkins server, run Grinder to validate your PR. Your can find recordings about how to use Grinder in AQA Lightning Talk Series: https://github.com/eclipse-openj9/openj9/wiki/AQA-Lightning-Talk-Series.  If you do not have access, the reviewers of your PR will run some tests.  Reviewers may ask you to run extra tests depending on what changes you have made in your PR.
```

> 8. Ensure the Grinder passes and provide the Grinder link in the PR comment. And ensure your changes passes auto PR builds that will be applied to your pull request.

Reviewer: ```suggestion
8. Ensure all related Grinder jobs pass and provide the Grinder links in the PR comment. Your changes must also pass the auto PR builds that will be applied to your pull request.
```

> 7. If you have permission, please run [Grinder](https://ci.adoptopenjdk.net/job/Grinder/) to validate your PR. Your can find recordings about how to use Grinder in AQA Lightning Talk Series: https://github.com/eclipse-openj9/openj9/wiki/AQA-Lightning-Talk-Series. If you do not have permission, you can skip this step. One of the reviwers will validate the PR for you during the review process.

Reviewer: ```suggestion
7. If you have permission, please run [Grinder](https://ci.adoptopenjdk.net/job/Grinder/) to validate your PR. Your can find recordings about how to use Grinder in AQA Lightning Talk Series: https://github.com/eclipse-openj9/openj9/wiki/AQA-Lightning-Talk-Series. If you do not have permission, you can skip this step. One of the reviewers will validate the PR for you during the review process.
```

> 9. Covert PR to `Ready for review` once the PR is ready.

Reviewer: ```suggestion
9. Convert PR to `Ready for review` once the PR is ready.
```
